FWCore/ServiceRegistry/interface/ModuleCallingContext.h

```diff
@@ -81,6 +84,7 @@ namespace edm {
     State state_;
   };
 
+  void exceptionContext(cms::Exception&, ModuleCallingContext const&);
```

Comment:
Where is this function called from? (I probably just missed it)

Reply:
Worker.cc

Reply:
Ah, this was a free function and not a member function. Thanks!